### ISSUE TITLE
Tags from Capture One are not separated

### ISSUE THREAD

**Issue by nicokaiser** [immich-app/immich#12403]
### The bug

Capture One seems to export keywords as `HierarchicalSubject` in a way different than what Lightroom does:

```xml
 <rdf:Description rdf:about=''
  xmlns:lr='http://ns.adobe.com/lightroom/1.0/'>
  <lr:hierarchicalSubject>
   <rdf:Bag>
    <rdf:li>Conference</rdf:li>
    <rdf:li>International conference</rdf:li>
    <rdf:li>JS Kongress 2017</rdf:li>
    <rdf:li>JSK17</rdf:li>
    <rdf:li>München</rdf:li>
    <rdf:li>Munich</rdf:li>
   </rdf:Bag>
  </lr:hierarchicalSubject>
 </rdf:Description>
```

Immich seems to interpret this as one tag: `Conference,International conference,JS Kongress 2017,JSK17,München,Munich`
I don't know if this should be saved differently by Capture One, but this behaviour leads to many very long Immich tags for every combination of exported keywords...

### The OS that Immich Server is running on

Debian

### Version of Immich Server

v1.114.0

### Version of Immich Mobile App

v1.113.1

### Platform with the issue

- [X] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
default docker-compose.yml
```

### Your .env content

```Shell
default .env
```

### Reproduction steps

1. Open a photo in Capture One
2. Apply multiple keywords to the photo
3. Export as JPEG
4. Import to Immich
5. The Photos has only one tag which contains all of the keywords at once

### Additional information

I can provide a full JPEG file and/or XMP if necessary

**Comment by smoltron** [comment]
The same happened to my 70000 photos. I ran metadata update to all and now the tags are not separated any more. What is strange that these tags are originally set by Lightroom. The result is that tags are not usable any more.

**Comment by jrasm91** [comment]
Sorry about that. This should be fixed with #12509.

**Comment by smoltron** [comment]
While it may be fixed, my database stays corrupted. How can we clean the faulty tags and recreate them again? Exctract metadata/All did not delete the faulty tags.

**Comment by dlang123** [comment]
A nice "Delete all empty tags" some place would possibly be nice for the future.

**Comment by jrasm91** [comment]
> A nice "Delete all empty tags" some place would possibly be nice for the future.

Yeah, we can definitely add something like this.

**Comment by nicokaiser** [comment]
> Yeah, we can definitely add something like this.

So this would be a new feature request, like "auto cleanup tags with no images"? Or can this be done in a trivial DB query?

**Comment by bo0tzz** [comment]
Already implemented in #12654
